modules/ezcontent_node/modules/ezcontent_smart_article/js/ezcontent_smart_article.js

```diff
@@ -11,21 +11,27 @@
    */
   Drupal.behaviors.ezcontent_smart_article = {
     attach: function (context, settings) {
-      $('.field--type-ezcontent-smart-image-tags .image-tag-field-wrapper .tag-wrapper li', context).click(function (e) {
+
+      // For Smart Image tags.
+      $('.image-tag-field-wrapper .tag-wrapper li', context).once().click(function () {
         var tagName = $(this).text();
-        var existingTags = $('.field--type-ezcontent-smart-image-tags .ui-autocomplete-input').val();
+        var imageWrapper = $(this).parent().parent().parent().parent();
```

Comment:
can we use it like this
```suggestion
       var imageWrapper = $(this).parents().eq(3);
```